app/utils/config.example.js

```diff
@@ -1,7 +1,8 @@
 
 module.exports = {
   GOOGLE_API_KEY: process.env.GOOGLE_API_KEY || 'AIzaSyCZxTONKtTHyz3qp-x4tJlH4c6VUKn9sd4',
-  ALGOLIA_INDEX_PREFIX: process.env.ALGOLIA_INDEX_PREFIX || 'development_service_Resource',
```

Comment:
You don't want to commit this part, will mess with other devs.

Reply:
Fixed